index.html

```diff
@@ -37,5 +37,177 @@ <h1 class="header">I'am Sentayhu <br>  Glad to see <br>you!</h1>
+     </ul>
+     <div class="discription">
+       <p class="discpara">A daily selection of 
+           privately personalized reads; no
+            accounts or sign-ups required.</p>
+     </div>
+     <ul class="connect-via">
+         <li class="lang">HTML</li>
+         <li class="lang">CSS</li>
+         <li class="lang">JavaScript</li>
+     </ul>
+     <button type="button" class="project1-btn">See Project</button>
+    </div>
+</div>
+<!-- ** END OF CARD 2 **-->
+<div class="subwork">
+    <div class="worksec1">
+    <img src="./assets/images/Snapshoot%20Portfolio2.svg" class="cardpic" alt="pic">
+    <div class="card"></div>
+    <h1 class="tonic1">Multi-Post Stories</h1> 
+     <ul class="connect-via card-info">
+         <li class="canopy">Canopy </li>
+     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters"> &nbsp;&nbsp; Back End Dev</li>
+     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters">&nbsp; &nbsp;2015</li>
+     </ul>
+     <div class="discription">
+       <p class="discpara">A daily selection of 
+           privately personalized reads; no
+            accounts or sign-ups required.</p>
+     </div>
+     <ul class="connect-via">
+         <li class="lang">HTML</li>
+         <li class="lang">CSS</li>
+         <li class="lang">JavaScript</li>
+     </ul>
+     <button type="button" class="project1-btn">See Project</button>
+    </div>
+</div>
+<!-- ** END OF CARD 3 **-->
+<div class="subwork">
+    <div class="worksec1">
+    <img src="./assets/images/Snapshoot%20Portfolio3.svg" class="cardpic" alt="pic">
+    <div class="card"></div>
+    <h1 class="tonic1">Multi-Post Stories</h1> 
+     <ul class="connect-via card-info">
+         <li class="canopy">Canopy </li>
+     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters"> &nbsp;&nbsp; Back End Dev</li>
+     <li class="list1"><img src="./assets/images/Counter.svg" class="counters" alt="counters">&nbsp; &nbsp;2015</li>
+     </ul>
+     <div class="discription">
+       <p class="discpara">A daily selection of 
+           privately personalized reads; no
+            accounts or sign-ups required.</p>
+     </div>
+     <ul class="connect-via">
+         <li class="lang">HTML</li>
+         <li class="lang">CSS</li>
+         <li class="lang">JavaScript</li>
+     </ul>
+     <button type="button" class="project1-btn">See Project</button>
+    </div>
+</div>
+</section>
+<br><br>
+<section  class="about">
+    <div>
+<h1 class="abouttxt"> About<br> Myself</h1>
+<p class="discpara">Hello I’m a software developer! I can help you 
+    build a product , feature or website Look through some of my work and experience! If you like what 
+    you see and have a project you need coded, don’t hestiate to contact me.</p>
+</div><br><br>
+<a class="lets-connect con" href="#">LETS CONNECT</a>
```

Comment:
- [ ] LET'S CONNECT should be inside a `<hx>/<h2>` tag instead of an anchor tag. Please fix it